Editor/Helper/AddressablesImportsMethodHandler.cs

```diff
@@ -0,0 +1,101 @@
+﻿namespace UnityAddressableImporter.Helper.Internal
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+    using System.Text.RegularExpressions;
+    using UnityEditor;
+    using UnityEngine;
+
+    public static class AddressablesImportMethodHandler
```

Comment:
This class could be renamed to AddressableImporterXXX (singular, import => importer) to keep consistent with other filenames.

Reply:
renamed to AddressablesImporterMethodHandler